testgrid/config.yaml

```diff
@@ -3333,6 +3366,24 @@ dashboards:
     alert_options:
       alert_mail_to_addresses: davanum+testgrid@gmail.com
 
+- name: conformance-cloud-provider-vsphere
+  dashboard_tab:
+  - name: periodic-v1.10
+    description: Runs conformance tests using kubetest against kubernetes v1.10 with cloud-provider-vsphere
+    test_group_name: ci-periodic-cloud-provider-vsphere-test-e2e-conformance-branch-v1.10
+    alert_options:
+      alert_mail_to_addresses: cnx@vmware.com
```

Comment:
Hi @figo,

Please also update the e-mail to `cnx+k8s-e2e@vmware.com`. The `+` is well-understood by e-mail servers to be metadata used to filter mail to one account.

Reply:
Hi @figo,

Let's use `cnx+e2e@vmware.com` to be safe. I'm not sure if you can have another separator after the `+`.